internal/mongodoc/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mongodoc_test // import "gopkg.in/juju/charmstore.v5-unstable/internal/mongodoc"
```

Comment:
You only need one canonical import path specifier per package, so let;'s not proliferate them everywhere, otherwise we have to update all of them when the package moves.